shells/backends/gcp_tf_backend/src/driver.py

```
@@ -75,9 +75,9 @@ def _validate_bucket_exists(self, bucket_name: str, context, logger):
             storage_client = storage.Client()
             get_bucket = storage_client.get_bucket(bucket_name)
             if len(str(get_bucket)) < 0:
-                raise ValueError(f"Bucket {bucket_name} not found")
+                logger.exception(f"Bucket {bucket_name} not found")
```

There is no "raise" in this case. You only write to log. Dont we need to raise an error here?